src/xo-app/host/tab-network.js

```diff
@@ -89,10 +85,25 @@ class ConfigureIpModal extends Component {
 @connectStore(() => ({
   vifsByNetwork: createGetObjectsOfType('VIF').groupBy('$network')
 }))
-class PifItem extends Component {
+
+class PifItemVlan extends Component {
+  _editPif = vlan =>
+    editPif(this.props.pif, { vlan })
+  render () {
+    const {pif} = this.props
```

Comment:
Try and keep spaces inside the brackets (`{ pif }`) when destructuring or creating objects. The goal is to try and keep everything as homogeneous as possible in the code even if there aren't explicit rules for everything, unfortunately :)